apps/server/src/modules/rocketchat/rocket-chat.service.ts

```diff
@@ -43,7 +43,7 @@ interface AdminIdAndToken {
 
 @Injectable()
 export class RocketChatService {
-	private adminIdAndToken: AdminIdAndToken;
+	private adminIdAndToken!: AdminIdAndToken;
```

Comment:
It seems the case, that this is indeed initialized lazy and can be undefined. Maybe we should use the question mark operator here?
```suggestion
	private adminIdAndToken?: AdminIdAndToken;
```